package.json

```diff
@@ -46,8 +46,8 @@
     "remark-parse": "4.0.0",
     "semver": "5.4.1",
     "string-width": "2.1.1",
-    "typescript": "2.5.3",
-    "typescript-eslint-parser": "git://github.com/eslint/typescript-eslint-parser.git#9c71a627da36e97da52ed2731d58509c952b67ae",
+    "typescript": "2.6.2-insiders.20171120",
```

Comment:
Isn't it a [TS 2.7 feature](https://github.com/Microsoft/TypeScript/wiki/Roadmap#27-january-2018)?

Reply:
They've backported to 2.6.2 (https://github.com/Microsoft/TypeScript/pull/19249#issuecomment-340896466, https://github.com/Microsoft/TypeScript/pull/20002)

Reply:
![image](https://user-images.githubusercontent.com/8341033/33312282-59107538-d462-11e7-9e61-e0d3a9d74c42.png)